python/GafferSceneUI/ShuffleAttributesUI.py

```diff
@@ -49,9 +49,9 @@
 	the filtered locations. The deleteSource plugs may be used to remove the original
 	source attribute(s) after the shuffling has been completed.
 
-	An additional context variable \"${source}\" can be used on the destination plugs
+	An additional context variable `${source}` can be used on the destination plugs
 	to insert the name of each source attribute. For example, to prefix all attributes
-	with \"user:\" set the source to \"*\" and the destination to \"user:${source}\").
+	with `user:` set the source to `*` and the destination to `user:${source}`.
```

Comment:
This one seems better as `render:` (or just `prefix:` again maybe)

Reply:
My intention at this point isn't to encourage users away from the existing convention. I think that will be easier to swallow when we can present a complete picture that is self-consistent, including the `render:arnold:*` etc stuff.